src/backend/chat_service/chat_service.py

```diff
@@ -141,13 +141,16 @@ async def list_files(request: Request, customer_guid: str):
             logger.info(f"No files found in bucket '{customer_guid}'")
 
         return {"files": file_list}
-    except Exception as e:
```

Comment:
YOu should always capture Base classException instead of a specific exception

Post capturing the exception you can use if else to specify the reason for exception